psycop/common/model_training_v2/trainer/preprocessing/steps/filters.py

```diff
@@ -1,6 +1,9 @@
+import re
```

Comment:
Perhaps we already want to split these filters into a row_filters file and a col_filters file 👍